.dockerignore

```diff
@@ -1,7 +1,7 @@
 Dockerfile
```

Comment:
The commit `feat: use Kompendium instead of Docz` should perhaps be called `docs: use Kompendium instead of Docz`? We don't really add anything to the published package…

Reply:
⚡